ssr-express/lib/createAmpSsrMiddleware.js

```diff
@@ -0,0 +1,107 @@
+/**
```

Comment:
Can we rename this file to AmpSsrMiddleware?

Reply:
done.